docs/release_notes_pdk.md

```diff
@@ -1,6 +1,31 @@
 # PDK release notes
 
 New features, enhancements, and resolved issues for the PDK 2.x release series.
+
+## PDK 2.1
+
+### New features and enhancements
+
+#### Add `pdk env` subcommand
+
+Adds an experimental `pdk env` subcommand, which prints the current environment variables used by the PDK (https://github.com/nkanderson).
+
+### Resolved issues
+
+#### `--verbose` option broken for  `pdk new defined_type`
+
+With [ddfreyne/cri#103](https://github.com/ddfreyne/cri/issues/103) merged, this issue has been resolved.
+
+#### PDK output scrollback in VS Code limited
+
+Resolved an issue using PDK in the integrated terminal of VS Code by disabling ansicon on Windows.
+
+### Template Changes and Fixes
+
+* Relocation of `inventory.yaml` file used by Litmus. This prevents accidental execution of acceptance tests in production systems. (https://github.com/pmcmaw)
```

Comment:
```suggestion
* Relocated the`inventory.yaml` file used by Litmus. This prevents accidental execution of acceptance tests in production systems. (https://github.com/pmcmaw)
```

Reply:
These template bullets have historically been in the past tense. I think that works better for these.